bundles/org.openhab.binding.modbus.kermi/src/main/resources/OH-INF/thing/xcenter-state-channel-types.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="modbus"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+	<channel-type id="global-state-channel">
+		<item-type>String</item-type>
+		<label>Global State</label>
+		<description>Current mode of your heat pump</description>
+	</channel-type>
+	<channel-type id="global-state-id-channel">
```

Comment:
What is the purpose of have two state channels that represent the same, they only have a different display. 
I would suggest to use one channel and combine options and translations